blueprint/index.md

```diff
@@ -1,224 +1,250 @@
+   ![Web Services Data Actions in Genesys Cloud Integrations Search](images/1AWebServicesDataActionInstall.png)
 
-![Rename and describe the Web Services Data Action](images/1BRenameWebServicesDataAction.png)
+2. Rename the web services data action and provide a short description.
 
-3. Navigate to **Configuration > Credentials** and click **Configure**.
+   ![Rename and describe the Web Services Data Action](images/1BRenameWebServicesDataAction.png)
 
-![Navigate to Configuration -> Credentials -> Configure](images/1CConfigurationCredentials.png)
+3. Navigate to **Configuration** > **Credentials** and click **Configure**.
 
-4. Create the following five fields, provide the desired values, and then click **OK**:
+   ![Navigate to Configuration -> Credentials -> Configure](images/1CConfigurationCredentials.png)
 
-- A: Set the client_id to the Application (client) ID from your Azure app.
-- B: Set the client_secret to the Client Secret from your Azure app.
-- C: Set the tenant_id to the Directory (tenant) ID from your Azure app.
-- D: Set the scope to https://graph.microsoft.com/.default.
-- E: Set the grant_type to client_credentials.
+4. For **User Defined (OAuth)** credential type, add the following five fields, enter the required values, and then click **OK**:
 
-![Create fields and values](images/1DFieldsandValues.png)
+   * A: Set the client_id to the Application (client) ID from your Azure app.
+   * B: Set the client_secret to the Client Secret from your Azure app.
+   * C: Set the tenant_id to the Directory (tenant) ID from your Azure app.
+   * D: Set the scope to https://graph.microsoft.com/.default.
+   * E: Set the grant_type to client_credentials.
 
-5. Import Authentication Data Action
+   ![Create fields and values](images/1DFieldsandValues.png)
 
-The Authentication data action is imported into another data action.  When a new Web Services Data Actions Integration is created within an org, a Custom Auth data action is also created.
+5. Import authentication data action
 
-![Custom Auth Data Action associated with Web Services Data Actions Integration](images/1ECustomAuthDataAction.png)
+   The authentication data action is imported into another data action. When you add a new web services data actions integration within an organization, a Custom Auth data action is created automatically.
+   Navigate to **Integrations** > **Actions** and open the **Custom Auth** data action.
 
-6. Open this Custom Auth data action.
-7. The *Office-365-Auth.customAuth.json* file provided in the [microsoft-teams-blueprint repo](https://github.com/MyPureCloud/microsoft-teams-blueprint "Opens the GitHub repo") on GitHub is the supplied Auth Data Action. Download this file and save it to your desktop for import into Genesys Cloud.
-8. Click **Import** and **Import Action** to import the supplied Auth Data Action.
+   ![Custom Auth Data Action associated with Web Services Data Actions Integration](images/1ECustomAuthDataAction.png)
```

Comment:
Caps style. In Genesys Cloud, we avoid title caps. Here are the full details: https://intranet.genesys.com/display/PureCloud/Doc+standards%3A+Capitalization